GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -295,6 +295,8 @@ namespace gdjs {
      * @returns The current instance for chaining.
      */
     fade(from: float, to: float, duration: float): this {
+      from = clampVolume(from);
+      to = clampVolume(to);
       if (this._id !== null) this._howl.fade(from, to, duration, this._id);
```

Comment:
This can be simplified:

```suggestion
      if (this._id !== null) this._howl.fade(clampVolume(from), clampVolume(to), duration, this._id);
```